resources/cassandra.yaml

```diff
@@ -144,6 +142,24 @@ spec:
                     values:
                       - cassandra
               topologyKey: kubernetes.io/hostname
+      securityContext:
+          runAsUser: 1000
+          runAsGroup: 1000
+          fsGroup: 1000
+      initContainers:
+        - name: fix-perms
+          image: quay.io/gravitational/debian-tall:0.0.1
```

Comment:
Tag `0.0.1` is not used anymore because it is pointing to Jessie. Use `buster` instead.